CMakeLists.txt

```diff
@@ -89,7 +89,6 @@ if (${CMAKE_BUILD_TYPE} STREQUAL DEBUG)
     # remove_definitions().
     #
     add_definitions(-fstack-check -fstack-protector)
-    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g")
```

Comment:
Seems to be ok:

cd /home/fermin/src/fiware-orion/BUILD_DEBUG/src/lib/parseArgs && /usr/bin/c++   -DDEBUG -DDEBUG_fermin -DLINUX -DLM_ON -DLONG_RAPIDXML_NAME_SPACE -Wall -Wno-unknown-pragmas -D_LARGEFILE64_SOURCE -D_GNU_SOURCE -Werror -fno-var-tracking-assignments -DLINUX -g -I/home/fermin/src/fiware-orion/src/lib    -fPIC -DDEBUG_SER  -fstack-check -fstack-protector -o CMakeFiles/pa.dir/paOptions.cpp.o -c /home/fermin/src/fiware-orion/src/lib/parseArgs/paOptions.cpp

Previous to this removal with have "...-DLINUX -g -g...".